.github/workflows/release.yml

```diff
@@ -118,33 +180,21 @@ jobs:
           name: moonbase-standalone
           path: build/standalone
 
-  ####### Checking generated Cargo files #######
-
-  check-cargo-toml:
-    runs-on: ubuntu-latest
-    needs: build
```

Comment:
I think the cargo.lock job worked as it was because of this

Reply:
I think the needs is only a condition to run or not the job but not to use its context (specially considering you can have multiple jobs in the needs, that would conflict with each other)

Reply:
I believe the job was executed and successful but it was simply not having any cargo file to check because the folder was empty.

Reply:
doc:
```
Each job runs in a fresh instance of the virtual environment specified by runs-on
```